ReactAndroid/src/main/res/devsupport/values/strings.xml

```diff
@@ -13,7 +13,7 @@
   <string name="catalyst_settings_title" project="catalyst" translatable="false">Catalyst Dev Settings</string>
   <string name="catalyst_jsload_title" project="catalyst" translatable="false">Please wait…</string>
   <string name="catalyst_jsload_message" project="catalyst" translatable="false">Fetching JS bundle</string>
-  <string name="catalyst_jsload_error" project="catalyst" translatable="false">Unable to download JS bundle. Did you forget to start the development server or connect your device?</string>
+  <string name="catalyst_jsload_error" project="catalyst" translatable="false">Unable to download JS bundle from the dev server.\n\nTry the following to fix the issue:\n• Ensure that the node server is running - run \'npm start\' from your project root\n• Ensure that your device/emulator is connected to your machine and has USB debugging enabled - run \'adb devices\' to see a list of connected devices\n• If you\'re on a device connected to the same machine, run \'adb reverse tcp:8081 tcp:8081\' to forward requests from your device\n• If your device is on the same Wi-Fi network, set \'Debug server host &amp; port for device\' in \'Dev settings\' to your machine\'s IP address and the port of the local dev server - e.g. 10.0.1.1:8081</string>
```

Comment:
@astreet Okies. I think I'll remove other changes from this PR until we figure out how to correctly check for bundle and have only the message improvement.

1) How do I distinguish between FB internal and OSS?
2) Will do
3) Will try

Also if you can provide me some guidance on how to correctly handle the error it'll be great. Currently for the first time it always says cannot find variable... error, which is of no help.